From: Ronnie Sahlberg <ronniesahlberg@gmail.com>
Date: Sat, 3 Jun 2017 11:33:51 -0700
Subject: [PATCH] Fix another compiler warning

We do need a default: arm here as the switch is not supposed to deal with
all possible values for the enum.

Signed-off-by: Ronnie Sahlberg <ronniesahlberg@gmail.com>
(cherry picked from commit fb45f0343e3de45f2bc7516591d36c7466c2184e)
---
 test-tool/iscsi-support.c | 19 +++++++++++--------
 1 file changed, 11 insertions(+), 8 deletions(-)

diff --git a/test-tool/iscsi-support.c b/test-tool/iscsi-support.c
index ed00ce7..3f860ed 100644
--- a/test-tool/iscsi-support.c
+++ b/test-tool/iscsi-support.c
@@ -2922,14 +2922,17 @@ void populate_ident_tgt_desc(unsigned char *buf, struct scsi_device *dev)
 
         for (desig = inq_di->designators; desig; desig = desig->next) {
                 switch (desig->designator_type) {
-                        case SCSI_DESIGNATOR_TYPE_VENDOR_SPECIFIC:
-                        case SCSI_DESIGNATOR_TYPE_T10_VENDORT_ID:
-                        case SCSI_DESIGNATOR_TYPE_EUI_64:
-                        case SCSI_DESIGNATOR_TYPE_NAA:
-                                if (prev_type <= desig->designator_type) {
-                                        tgt_desig = desig;
-                                        prev_type = desig->designator_type;
-                                }
+                case SCSI_DESIGNATOR_TYPE_VENDOR_SPECIFIC:
+                case SCSI_DESIGNATOR_TYPE_T10_VENDORT_ID:
+                case SCSI_DESIGNATOR_TYPE_EUI_64:
+                case SCSI_DESIGNATOR_TYPE_NAA:
+                        if (prev_type <= desig->designator_type) {
+                                tgt_desig = desig;
+                                prev_type = desig->designator_type;
+                        }
+                        continue;
+                default:
+                        continue;
                 }
         }
         if (tgt_desig == NULL) {
